### DISCUSSION TITLE
关于ML域名的相关通知

### DISCUSSION THREAD

**Discussion by fxzxmic** [luolongfei/freenom#232] [General]
详见：https://nic.ml/
已经被马里政府接管。

另外吐槽一句，我就说了一句.co续费很贵就被管理员踢了？不解释下吗？
https://t.me/freenom_auto_renew
@luolongfei

**Comment by luolongfei** [comment]
机器人管理员有误封的情况，已经禁用它的 ban 人权限

**Comment by luolongfei** [comment]
可以私信我给你解封

**Reply by fxzxmic** [reply]
你用户名是啥？要么你解封我吧，我用户名和GitHub一样。🫥

**Reply by luolongfei** [reply]
@fxzxmic 现在试试能不能重新加入
